conans/client/graph/graph_builder.py

```diff
@@ -288,7 +288,8 @@ def _expand_require(self, require, node, graph, check_updates, update, remotes,
                 # Maybe it was an ALIAS, so we can check conflict again
                 conflict = self._conflicting_references(previous, require.ref, node.ref)
                 if conflict:
-                    raise ConanException(conflict)
+                    raise ConanException("Conflicting references: {} => {}".format(previous,
```

Comment:
`node.ref` is None so no Exception was raised inside `self._conflicting_references` (this is a bit weird desing).